packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -62,10 +62,10 @@ class _FutureMemoryImage extends ImageProvider<_FutureMemoryImage> {
     return SynchronousFuture<_FutureMemoryImage>(this);
   }
 
+  // ignore:deprecated_member_use
   /// See [ImageProvider.load].
```

Comment:
This *comment* line was the problem (square brackets ImageProvider.load reference counting as a "use" !!!)...

Reply:
wow